examples/sagemaker_orchestration/README.md

```diff
@@ -0,0 +1,159 @@
+# 🏃 Run pipelines in production using Sagemaker
+
+[Sagemaker Pipelines](https://aws.amazon.com/sagemaker/pipelines?tag=soumet-20)
+is a serverless ML workflow tool running on AWS. It is
+an easy way to quickly run your code in a production-ready, repeatable 
+cloud orchestrator that requires minimal setup without provisioning and paying 
+for standby compute.
+
+## 📄 Pre-requisites
+
+In order to run on Sagemaker you will need to do some configuration
+within AWS to create all the required resources and permissions. In total, you will need to have:
+
+- [ECR container registry](https://docs.zenml.io/component-gallery/container-registries/amazon-ecr)
+- [S3 artifact store](https://docs.zenml.io/component-gallery/artifact-stores/amazon-s3)
+- [AWS Secrets Manager](https://docs.zenml.io/component-gallery/secrets-managers/aws).
```

Comment:
I think the secrets manager isn't even needed in this example?

Reply:
Needed for the artifact store authentication.

Reply:
I guess in theory could be solved using AWS roles as well, but doesn't hurt to have it here